qemu/tests/qemu_guest_agent.py

```diff
@@ -693,41 +660,58 @@ def find_interface_by_name(interface_list, target_interface):
                        interfaces_pre_add]
         if (len(bridge_list) != 1) or \
            ("lo_brige" != bridge_list[0]["name"]):
-            error.TestFail("the interface list info after interface was down"
-                           " was not as expected")
+            test.fail("the interface list info after interface was down"
+                      " was not as expected")
 
-    @error.context_aware
     def _action_before_fsfreeze(self, *args):
         session = self._get_session(self.params, None)
         self._open_session_list.append(session)
 
-    @error.context_aware
     def _action_after_fsfreeze(self, *args):
-        error.context("Verfiy FS is frozen in guest.")
+        logging.info("Verfiy FS is frozen in guest.")
         if not isinstance(args, tuple):
             return
 
         if not self._open_session_list:
-            raise error.TestError("Could not find any opened session")
+            self.test.error("Could not find any opened session")
         # Use the last opened session to send cmd.
         session = self._open_session_list[-1]
         try:
             session.cmd(self.params["gagent_fs_test_cmd"])
         except aexpect.ShellTimeoutError:
             logging.debug("FS freeze successfully.")
         else:
-            raise error.TestFail("FS freeze failed, guest still can"
-                                 " write file")
+            self.test.fail("FS freeze failed, guest still can"
+                           " write file")
+
+    def gagent_check_reset_reboot_shutdown(self, test, params, env):
+
+        """
+        Execute "system_reset" and send "shutdown,reboot" command to guest agent after FS freezed
```

Comment:
Make sure each line does not exceed 90 char

Reply:
@vivianQizhu Thanks,  I have updated a comments.  can you review it again?